### DISCUSSION TITLE
AWS stopped working

### DISCUSSION THREAD

**Discussion by mertf** [coccoinomane/crabada.py#69] [Q&A] [answered]
This question is not directly related to the bot but I am hoping to get some insight. 

I have set up an AWS instance for my cronjobs, and it had been working great until just a few hours ago. I am unable to connect via SSH to the instance, and it is giving me two errors or warnings. One says ```client_loop: send disconnect: Broken pipe``` and the other is ```operation timed out```. I couldn't figure out why all of a sudden everything would stop working. Could you guys point me in the right direction? Or at the very least, show me where to look?

**Comment by coccoinomane** [comment, marked as answer]
Hi @mertf !

I can't go into specifics because I am running on Vultr, but this seems like a problem on AWS' side: maybe a downtime? You could try opening a ticket.

If instead it is another kind of issue that needs to be fixed, to be honest, I would just spin up a Vultr machine at $6/month, given how quick it is 😅

Cheers,
Cocco

**Reply by mertf** [reply]
Hi @coccoinomane,

Thank you for your suggestion. I was just about to spin up a Vultr machine, but I couldn't just let AWS go without obsessively figuring out the reason to the issue. I finally did, and wanted to share here :) 

I have multiple wallets with multiple teams. So I set up different folders just for looting reinforcements and closing them. That resulted in running the machine out of memory. I upgraded it to the next level up, and it is working now.

**Reply by coccoinomane** [reply]
Thank you @mertf for sharing your experience!

Good to know about the memory usage: I have never had the issue because I only have one team 😅

This should be fixed with multi-user support: if you are interested in the feature, feel free to voice your support in https://github.com/coccoinomane/crabada.py/issues/62.

Cheers,
Cocco
